packages/apps/src/SideBar/index.tsx

```diff
@@ -28,9 +28,10 @@ interface Props {
   isCollapsed: boolean;
   isMenuOpen: boolean;
   toggleMenu: () => void;
+  isAdvance: boolean;
 }
 
-function SideBarContainer ({ className, collapse, handleResize, isCollapsed, isMenuOpen, toggleMenu }: Props): React.ReactElement<Props> {
```

Comment:
Can you plz revert the changes in this file, otherwise, it gonna have conflict with the SideBar PR?

Reply:
I will let it be.. we can first merge your changes, I will resolve the conflict.

Reply:
Rebased with ur changes..